src/libraries/Microsoft.Extensions.Logging.Console/ref/Microsoft.Extensions.Logging.Console.cs

```diff
@@ -62,6 +62,8 @@ public partial class ConsoleLoggerOptions
         [System.ObsoleteAttribute("ConsoleLoggerOptions.IncludeScopes has been deprecated. Use ConsoleFormatterOptions.IncludeScopes instead.")]
         public bool IncludeScopes { get { throw null; } set { } }
         public Microsoft.Extensions.Logging.LogLevel LogToStandardErrorThreshold { get { throw null; } set { } }
+        public int MaxQueueLength { get { throw null; } set { } }
+        public Microsoft.Extensions.Logging.Console.ConsoleLoggerBufferFullMode BufferFullMode { get { throw null; } set { } }
```

Comment:
I believe the approved API has 

```C#
public ConsoleLoggerQueueFullMode QueueFullMode { get; set; }
```

This uses "queue" and "buffer". We should just have one name.